client/components/ItemIcon.jsx

```diff
@@ -67,20 +69,25 @@ export const ItemIcon = ({item, big, white, blue, showRepeating}) => {
             return null;
         }
 
+
         if (!showRepeating || !multiValidator(item)) {
+            title = gettext(title);
+
             return big ?
-                (<span className="double-size-icn double-size-icn--light">
+                (<span title={title} className="double-size-icn double-size-icn--light">
```

Comment:
Some details are displayed as a 'span' with a white background where as some are ui-framework's tooltips. Should we be having consistent behavior across the entire module ?